drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/test/java/org/drools/workbench/screens/scenariosimulation/kogito/client/converters/scesim/ApiJSInteropConverterTest.java

```diff
@@ -0,0 +1,5 @@
+package org.drools.workbench.screens.scenariosimulation.kogito.client.converters.scesim;
+
+public class ApiJSInteropConverterTest {
```

Comment:
It should be nice a test that verify that all fields of each bean is invoked by each mapper so we can easily track changes in the future. For example spy all getters from `Background` check if all of them are invoked. I think you might need to use reflection for that

Reply:
@danielezonca Unfortunately this class is not testable. This is because every converter method instantiate a new JSInterop object and this generates a `java.lang.UnsatisfiedLinkError` when you try to call the converter methods. For the same reason, I don't see a way to mock these Objects.